analyses/cms-open-data-ttbar/ttbar_analysis_pipeline.py

```diff
@@ -76,16 +76,17 @@
 #
 # | setting | number of files | total size |
 # | --- | --- | --- |
-# | `1` | 9 | 16.3 GB |
-# | `5` | 45 | 81.7 GB |
-# | `10` | 86 | 157 GB |
-# | `50` | 357 | 678 GB |
-# | `100` | 590 | 1.09 TB |
-# | `500` | 1542 | 2.58 TB |
-# | `1000` | 2249 | 3.57 TB |
-# | `-1` | 2269 | 3.59 TB |
+# | `1` | 12 | 25.1 GB |
```

Comment:
Did the splitting into categories change, or why is the minimum number of files for setting `1` changing from 9 to 12?

Reply:
for single_top_tW, for nanoAOD I have scaledown, scaleup, and DS in addition to nominal. Should I remove these?

Reply:
Yes, let's remove these as they are currently not being used. I'll track the paths via #118 but this is low priority to include again in the pipeline.